cheatsheets/DotNet_Security_Cheat_Sheet.md

```diff
@@ -14,6 +14,13 @@ Individual frameworks can be kept up to date using [NuGet](http://nuget.codeplex
 
 Remember that third-party libraries have to be updated separately and not all of them use NuGet. ELMAH for instance, requires a separate update effort.
 
+## Security Announcements
+
+Receive security notifications by selecting the "Watch" button at the following repositories:
+
+- [.NET Core Security Announcements](https://github.com/dotnet/announcements) 
```

Comment:
Should this not link to the following [security bulletin](https://github.com/dotnet/announcements/issues?q=is%3Aopen+is%3Aissue+label%3ASecurity) since we are talking about security announcements?

Reply:
@ThunderSon,
Yikes!  You are right!   I forgot to add the security label and "isopen" for the .NET Core link.   Thanks so much for catch!

Reply:
I corrected the problem in patch-1.  Thanks again.